keps/sig-node/3008-qos-class-resources/README.md

```diff
@@ -472,6 +451,26 @@ usage of container-level QoS-class resources.
 
 Just using LimitRanges for specifying defaults could simplify the API.
 
+#### Implicit defaults
+
+If nothing is requested for a QoS-class resource, a pod/container still
+implicitly belongs to some class. By design there is no such thing as being in
+no QoS class. What this "unnamed" default class means or how it is handled is
+considered an implementation detail of the runtime. However, it would be
+potentially desirable for the user to be able to see what was classes the
+application was implicitly assigned to (even if nothing was explicitly
+requested). The first implementation phase contains no mechanism for this.
+
+Some considerations/questions:
+
+- where can user see what was implicitly given? new field in PodStatus?
+- how to communicate what is effective for each pod and container
+- implicit default might change after runtime re-configuration
```

Comment:
you can't change the default from underneath a running pod - this is why persisting it into status is important

Reply:
Why is it impossible to change for a running pod? Is it a limitation we need to enforce or the limitation of specific resource types?

Reply:
I'd have the same question as @SergeyKanzhelev. I think it could change for a running pod (if it's possible for the specific resource type). In some case container runtime would just e.g. adjust the throttling parameters according to the new default